landing-page/components/Carousel.tsx

```diff
@@ -0,0 +1,28 @@
+export const Carousel = () => {
+  return (
+    <div id="carouselExampleCrossfade" className="carousel slide carousel-fade relative" data-bs-ride="carousel">
+      <div className="carousel-inner relative w-full overflow-hidden">
+        <div className="carousel-item  float-left w-full">
+          <div className="flex h-96" data-carousel-item="active">
```

Comment:
`id="carouselExampleCrossfade"` as well as some of the classnames aren't used for anything -> remove

(this applies to other files as well)